H2MLauncher.UI/ViewModels/ServerFilterViewModel.cs

```diff
@@ -69,7 +124,7 @@ private bool ApplyTextFilter(ServerViewModel server)
 
             string lowerCaseFilter = FilterText.ToLower();
 
-            if (!server.ToString().Contains(lowerCaseFilter, StringComparison.OrdinalIgnoreCase))
+            if (!server.HostName.ToString().Contains(lowerCaseFilter, StringComparison.OrdinalIgnoreCase))
```

Comment:
Why not keep it filtering the IP too?

Reply:
As that is not what people expect. People expect it to be on the name of the server.